00-HTML-CSS-basics/ex 2/2.5/2.5.1/css/style.css

```diff
@@ -0,0 +1,64 @@
+*,
+*:before,
+*:after{
```

Comment:
Nice grouping, another thing to get this clean try to leave one white space between the css selector definition and the curly brace :)